inc/Engine/Optimization/UrlTrait.php

```diff
@@ -3,6 +3,8 @@
 
 namespace WP_Rocket\Engine\Optimization;
 
+use WP_Rocket\Logger\Logger;
```

Comment:
I don't see where we are actually using Logger in here. Accedental IDE auto-insertion, maybe?

Reply:
this was added during my investigation of the issue, good catch.

Done